girder/utility/plugin_utilities.py

```diff
@@ -88,6 +88,22 @@ def loadPlugins(plugins, root, appconf, apiRoot=None, curConfig=None):
     return root, appconf, apiRoot
 
 
+def getPluginParentDir(name, curConfig=None):
+    """
+    Finds the directory a plugin lives in and returns it. This throws
+    an exception if it can't find a directory named name in any of the
+    set plugin_directory paths.
+
+    :params name: The name of the plugin (i.e. its directory name)
```

Comment:
Do we want to make the equivalency here of the plugin name to the plugin directory?  The `plugin.yml` gives a name that doesn't have to be the same as the directory.  I know I have run into confusion in the past when they were named differently.  This may be an issue more generally in girder; we should make sure the usage is consistent.